.github/workflows/build-test-release.yml

```diff
@@ -150,36 +150,70 @@ jobs:
       fail-fast: false
       matrix:
         splunk-version: ["8.1","8.2"]
+    env:
+      KUBECONFIG: /etc/rancher/k3s/k3s.yaml
     steps:
       - uses: actions/checkout@v2
         with:
           submodules: true
+      - uses: actions/setup-python@v1
```

Comment:
Is there a reason to use `v1` of `actions/setup-python`?  There is v3 available: https://github.com/actions/setup-python/releases/tag/v3.0.0

Reply:
Updated the action setup-python to v3.